04_numeronym/CameronSchafer/main_test.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// main() test
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("[a11y K8s abc]\n")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()\nexpected: %q\nactual: %q", expected, actual)
+	}
```

Comment:
Since you're already using testify, is there any reason you're not using `assert.Equal` here? If you want this to be a fatal error, you can use the [require package](http://godoc.org/github.com/stretchr/testify/require).

Reply:
Updated to use assert.Equalf